packages/tooltips/examples/default.md

```diff
@@ -0,0 +1,100 @@
+The `Tooltip` component acts as a high-abstraction API over the
+[useTooltip()](https://www.npmjs.com/package/@zendeskgarden/container-tooltip)
+hook.
+
+All props passed to the root element are proxied onto the tooltip element.
+
+```jsx
+const { Button } = require('@zendeskgarden/react-buttons/src');
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { Field, Label, Toggle } = require('@zendeskgarden/react-forms/src');
+const {
+  Dropdown,
+  Select,
+  Field: SelectField,
+  Label: SelectLabel,
+  Menu,
+  Item
+} = require('@zendeskgarden/react-dropdowns/src');
+
+initialState = {
+  placement: 'top',
+  showArrow: true,
+  type: 'dark'
+};
+
+const StyledSpacer = styled.div`
+  margin-bottom: ${props => props.theme.space.sm};
+`;
+
+<Grid>
+  <Row>
+    <Col md={4}>
+      <Well recessed>
+        <StyledSpacer>
+          <Dropdown selectedItem={state.placement} onSelect={placement => setState({ placement })}>
+            <SelectField>
+              <SelectLabel>Placement</SelectLabel>
+              <Select small>{state.placement}</Select>
+            </SelectField>
+            <Menu small>
+              <Item value="auto">auto</Item>
+              <Item value="top">top</Item>
+              <Item value="top-start">top-start</Item>
+              <Item value="top-end">top-end</Item>
+              <Item value="bottom">bottom</Item>
+              <Item value="bottom-start">bottom-start</Item>
+              <Item value="bottom-end">bottom-end</Item>
+              <Item value="end">end</Item>
+              <Item value="end-start">end-start</Item>
+              <Item value="end-bottom">end-bottom</Item>
+              <Item value="start">start</Item>
+              <Item value="start-top">start-top</Item>
+              <Item value="start-bottom">start-bottom</Item>
+            </Menu>
+          </Dropdown>
+        </StyledSpacer>
+        <StyledSpacer>
+          <Field>
+            <Toggle
+              checked={state.showArrow}
+              onChange={e => setState({ showArrow: e.target.checked })}
+            >
+              <Label>Arrow</Label>
+            </Toggle>
```

Comment:
Is there a way to keep the focus on the underlying input element for `<Toggle>` after it is triggered? 

Currently, space keypress causes the toggle to tigger, then puts the focus on the `<body>` tag. A keyboard user can select the toggle once via space key, but the second space press will cause the browser to scroll to the bottom of the page since focus is now on the `<body>`. 

This can be reproduced by selecting the `<Toggle>` twice with a space key:

![boom2](https://user-images.githubusercontent.com/1811365/68904744-7d772000-06f3-11ea-9772-a7fd2667826b.gif)

Reply:
This is due to `stylguidist` updating it's global example state. We have a global `State` helper component which I've used to stop the example from re-rendering.

Reply:
> We have a global State helper component which I've used to stop the example from re-rendering.

Thank you for the explanation, that's very helpful to know about the `State` helper 🙏.  Checked again and the example is working great.